src/Insect/Parser.purs

```diff
@@ -218,7 +218,7 @@ normalUnitDict = Dictionary
   , Q.candela ==> ["candela", "cd"]
   , Q.watt <> Q.hour ==> ["Wh"]
   , Q.watt ==> ["watts", "watt", "W"]
-  , Q.byte ==> ["Bytes", "bytes", "Byte", "byte", "B"]
+  , Q.byte ==> ["Bytes", "bytes", "Byte", "byte", "B", "Octet", "octet", "Octets", "octets", "o"]
```

Comment:
Unfortunatly, this does not pass the unit tests. The ordering is crucial here. The parser tries all units from top to bottom, left to right. If `octet` comes before `octets`, it will successfully parse `octet` for an input of `octets` and subsequently fail because of the trailing `s` (and therefore recognize the input as an identifier `octets`). So the plural versions should come first.

Also, if possible, I'd like to avoid reserving single letter identifiers (`o`) for units because that means that the variable `o` cannot be used. I don't think we need `o` here as it does not seem to be used frequently and there is already `B` for a shortcut.